### DISCUSSION TITLE
'@rollup/plugin-babel' issue

### DISCUSSION THREAD

**Discussion by joamatab** [fief-dev/fief#448] [Q&A]
'@rollup/plugin-babel'

![image](https://github.com/user-attachments/assets/5e8cefd6-529b-4af6-ba6f-6fe5ac3e2a28)

Im using python3.12 and npm@10.1.0 /home/j/.nvm/versions/node/v20.9.0/lib/node_modules/npm

**Comment by fief-bailiff** [comment]
Hail, @joamatab 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Did you run `npm install`? 😊
